src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Protocol/DelegatedTypes.cs

```diff
@@ -39,6 +39,11 @@ internal record DelegatedValidateBreakpointRangeParams(
     [property: JsonPropertyName("projectedRange")] LspRange ProjectedRange,
     [property: JsonPropertyName("projectedKind")] RazorLanguageKind ProjectedKind) : IDelegatedParams;
 
+internal record DelegatedDataTipRangeParams(
+    [property: JsonPropertyName("identifier")] TextDocumentIdentifierAndVersion Identifier,
+    [property: JsonPropertyName("projectedPosition")] Position ProjectedPosition,
```

Comment:
Nit: A little weird to have just a position while the type is called Range. We never map a range, always a position?

Reply:
The position is for the input param, the output is a data structure that has a couple ranges. The name here is based on the lsp message name, I'm flexible to change it if you prefer, I just thought that was the general form used here.

Reply:
No, I'm happy with the name as is, just had to double-check myself so figured I'd mention it, but certainly a nit.